db/seeds.rb

```diff
@@ -5,3 +5,44 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create!([
+  {
+    name: 'Michael'
+  },
+  {
+    name: 'Joshua'
+  },
+  {
+    name: 'Luke'
+  },
+  {
+    name: 'Mark'
+  },
+  {
+    name: 'Mathew'
+  },
+  {
+    name: 'James'
+  },
+  {
+    name: 'Peter'
+  },
+  {
+    name: 'John'
+  }
+])
+
+
+
+user1 = User.find_by(name: 'John')
+user1.events.create!({
```

Comment:
I had an error I this place when running `rails db:setup`
```rb
...
-- create_table("users", {:force=>:cascade})
   -> 0.1121s
rails aborted!
NoMethodError: undefined method `events' for #<User:0x00000000056adda8>
/home/maelfosso/.rvm/gems/ruby-2.6.5/gems/activemodel-5.2.4.3/lib/active_model/attribute_methods.rb:430:in `method_missing'
/home/maelfosso/Documents/Microverse/TSE/RoR/private-events/db/seeds.rb:38:in `<main>'
...
```

From your `User` model, there is not a relationship called `events`.